### ISSUE TITLE
business rules for marking "Complete" not being enforced for collaborators

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#1082]
NOTE - this bug doesn't need to hold up the release of notifications to live

**Describe the bug**
Read/write collaborator on live site (2.12.5) was able to mark an event as "complete" even though there was no end date. This is a violation of the business rules. See https://whispers.usgs.gov/event/200826 

**To Reproduce**
I tested this on the test.cr site and, as a partner user who was a read/write collaborator on someone else's event, I was able to mark 170902 complete, even though it has no end date, without a hitch.

If I log in as jchipault (event owner) and try to mark it as complete, I get this (as expected):
![image](https://user-images.githubusercontent.com/49287076/97018386-b4190580-1514-11eb-86b1-370c52d54b04.png)

It looks like the system is running some of the business rules in that if I'm logged in as the partner user collaborator and try to mark an event "complete" without an end date AND without a number for every species, I get the warning below (as expected). So I'm not sure which rules are run and which aren't, but they aren't all running 🙁
![image](https://user-images.githubusercontent.com/49287076/97018458-c85d0280-1514-11eb-8250-07debc585e37.png)

**Expected behavior**
Expect same business rule checks to be run when a read/write collaborator marks an event as "complete" as when the event owner marks an event as complete.

**Comment by aaronstephenson** [comment]
@JChipault is this still an issue?

**Comment by JChipault** [comment]
yup, was just able to reproduce as abilenetest on event 170873 on the test site.

**Comment by aaronstephenson** [comment]
This is a back end issue, transferring.
